openmls/src/schedule/errors.rs

```diff
@@ -30,14 +30,12 @@ pub enum KeyScheduleError {
 /// PSK secret error
 #[derive(Error, Debug, PartialEq, Clone)]
 pub enum PskSecretError {
+    #[error(transparent)]
+    LibraryError(#[from] LibraryError),
```

Comment:
The errors in here appear to be in `WelcomeError` as well. Are they used somewhere else or could we re-use them?

Reply:
Yes, and they are mapped to the ones in `WelcomeError`. I want to avoid having nested errors in the public errors as much as possible and therefore exposed the variants explicitly.